### PR TITLE
Add 3rd party metadata

### PR DESCRIPTION
**Pull request by sophieyfang**

### CODE REVIEW

**Review by qingling128 — APPROVED** [review]
LGTM

File: integration_test/third_party_apps_data/applications/apache/metadata.yaml [hunk 1]
```diff
@@ -1,3 +1,29 @@
...
+
+<pre>
+Total Accesses: 2
+Total kBytes: 1
+BusyWorkers: 1
+IdleWorkers: 4
+</pre>
+
+If you get a `404 Not Found` page instead, then the `mod_status` plugin is not
+enabled."
```
[the remark this answers is not in the document]
Comment (jefferbrecht) [reply integration_test/third_party_apps_data/applications/apache/metadata.yaml:26]: I don't have a strong preference but `|-` only strips the final `\n` which seems desirable IMO. Either way works.

[the remark this answers is not in the document]
Comment (sophieyfang) [reply integration_test/third_party_apps_data/applications/apache/metadata.yaml:26]: https://github.com/GoogleCloudPlatform/ops-agent/pull/519

File: integration_test/third_party_apps_data/applications/redis/metadata.yaml [hunk 2]
```diff
@@ -1,3 +1,6 @@
+public_url: "https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/redis"
+short_name: redis
+long_name: Redit
```
[the remark this answers is not in the document]
Comment (sophieyfang) [reply integration_test/third_party_apps_data/applications/redis/metadata.yaml:3]: See https://github.com/GoogleCloudPlatform/ops-agent/pull/519 :)

**Pull request merged**